analytics/index.yaml

```diff
@@ -24,7 +24,7 @@ paths:
         - name: idsite
           in: query
           #TODO: Make sure we're already using new name in docs.
-          description: Application ID (previously Website ID).
+          description: **Mandatory to track** Application ID (previously Website ID).
```

Comment:
I tried to indicate that it's mandatory by using `required` attribute. Is it enough or are there some other request types that can omit this value?

Reply:
It is always mandatory.